src/operations/indexes.ts

```diff
@@ -73,6 +71,13 @@ export type IndexSpecification = OneOrMore<
   | Map<string, IndexDirection>
 >;
 
+/** @public */
+export interface IndexInformationOptions {
+  full?: boolean;
+  readPreference?: ReadPreference;
```

Comment:
Some docs at least for `full`?

Reply:
sure.  but fwiw, it's not a new option 😓